acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/Signer.java

```diff
@@ -44,26 +44,30 @@ public Signer(final SignerConfiguration signerConfig, final NodeConfiguration no
             new HttpService(signerConfig.url()),
             signerConfig.pollingInterval().toMillis(),
             Async.defaultExecutorService());
-    this.transactions = new Transactions(jsonRpc);
+    this.transactions = new Transactions(this.jsonRpc);
   }
 
   public void start() {
     LOG.info("Starting EthSigner");
-    runner.start("EthSigner");
+    this.runner.start("EthSigner");
   }
 
   public void shutdown() {
     LOG.info("Shutting down EthSigner");
-    runner.shutdown();
+    this.runner.shutdown();
   }
 
   public Transactions transactions() {
-    return transactions;
+    return this.transactions;
+  }
+
+  public Web3j jsonRpc() {
```

Comment:
Don't expose the Web3j here, use `transactions` instead (which _may or may not_ yet exist on master 🤔 )

Reply:
How am I supposed to get the ethAcountsRequest from transactions?